src/library/components/page-header.js

```diff
@@ -149,106 +132,103 @@ var PageHeader = Component({
-      headerItems
+    return (
+      <ul className={'portal-pages-main-nav-contain'}>
+      {headerItems}
+      </ul>
     )
   },
 
   renderHeader: function () {
-    var navLinks = ''
+    let navLinks = ''
     if (this.state.windowWidth > 950 || !this.state.nav_menu_collapsed) {
       navLinks = this.renderNavLinks()
     }
-    var logoClass = this.state.logo_class
-    return div({className: 'theme-' + this.state.theme},
-      div({className: 'portal-pages-umbrella'},
-        div({className: 'portal-pages-umbrella-contain cols'},
-          div({className: 'portal-pages-concord-link col-12'},
-            a({href: 'https://concord.org/', className: 'portal-pages-concord-link__item'},
-              'Learn about the Concord Consortium ',
-              i({className: 'icon-arrow-diagonal'}, '')
-            )
-          )
-        )
-      ),
-      nav({className: 'concord-navigation cols no-collapse'},
-        div({className: 'logo-contain col-3'},
-          a({href: '/', title: 'Go to the Home Page'},
-            div({className: logoClass},
-              'Home'
-            )
-          )
-        ),
-        div({className: 'portal-pages-main-nav col-9'},
-          navLinks,
-          div({className: 'mobile-nav-contain'},
-            div({className: 'mobile-nav-btn'},
-              span({className: 'opener'}, 'Menu'),
-              span({className: 'closer'}, 'Close'),
-              div({className: 'mobile-nav-icon', onClick: this.handleNavMenuToggle},
-                span(),
-                span(),
-                span()
-              )
-            )
-          )
-        )
-      )
+    let logoClass = this.state.logo_class
+    let logoText = 'Home'
+    if (this.state.theme === 'learn') {
+      logoText = 'STEM Resource Finder'
+    }
+    return (
+      <div className={'theme-' + this.state.theme}>
+        <div className={'portal-pages-umbrella'}>
+          <div className={'portal-pages-umbrella-contain cols'}>
+            <div className={'portal-pages-concord-link col-12'}>
+              <a href="https://concord.org/" className={'portal-pages-concord-link__item'}>Learn about the Concord Consortium <i className={'icon-arrow-diagonal'}></i></a>
+            </div>
+          </div>
+        </div>
+        <nav className={'concord-navigation cols no-collapse'}>
+          <div className={'logo-contain col-3'}>
+            <a href="/" title="Go to the Home Page">
+              <div className={logoClass}>
+                <div className={'concord-logo__linktext'}>
+                  {logoText}
+                </div>
+              </div>
+            </a>
+          </div>
+          <div className={'portal-pages-main-nav col-9'}>
+            {navLinks}
+            <div className={'mobile-nav-contain'}>
+              <div className={'mobile-nav-btn'}>
+                <span className={'opener'}>{'Menu'}</span>
+                <span className={'closer'}>{'Close'}</span>
```

Comment:
Here are a couple more things that could be simplified.

Also I don't know if it is worth changing everything, but it seems you could also do `className="closer"` instead of `className={'closer'}`

Reply:
Ah, OK. Thanks. I'm pretty sure I got all of that in this latest commit.